internal/commands/app/create_inputs_test.go

```diff
@@ -209,14 +210,52 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Equal(t, "new-app", inputs.LocalPath)
 	})
 
-	t.Run("should error when path specified is another realm app", func(t *testing.T) {
+	t.Run("should create default local directory name when ui is set to auto confirm", func(t *testing.T) {
 		profile, teardown := mock.NewProfileFromTmpDir(t, "app_create_test")
 		defer teardown()
 
-		specifiedDir := "test-dir"
-		inputs := createInputs{LocalPath: specifiedDir}
+		_, _, ui, err := mock.NewVT10XConsoleWithOptions(
```

Comment:
[question] would `mock.NewConsoleWithOptions` suffice here?  doesn't look we test anything interactive here